docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,81 @@
+Pylake 1.0.0
+============
+
+.. only:: html
+
+Pylake has made it to `v1.0.0`! And with it, several new features and improvements have been added.
+
+Documentation
+-------------
+
+.. image:: docs.png
+
+The documentation has been completely reworked.
+There is now a new :doc:`theory section</theory/index>` which delves into the theory of the underlying data analysis methods, including examples and references to literature for further reading.
+This information has been separated from the :doc:`tutorial section</tutorial/index>` so the tutorials can focus specifically on teaching you how to use Pylake.
```

Comment:
~~~suggestion
It was split out from the :doc:`tutorial section</tutorial/index>` so that the tutorials can focus more on how to use Pylake.
~~~